features/user-reset-password.feature

```diff
@@ -41,3 +41,40 @@ Feature: Reset passwords for one or more WordPress users.
       """
       {ORIGINAL_PASSWORD}
       """
+
+  @require-wp-4.3
+  Scenario: Reset the password of a WordPress user, and show the new password
+    Given a WP installation
+
+    When I run `wp user get 1 --field=user_pass`
+    Then save STDOUT as {ORIGINAL_PASSWORD}
+
+    When I run `wp user reset-password 1 --skip-email --show-password`
+    Then STDOUT should contain:
+      """
+      Password:
+      """
+    And an email should not be sent
+
+  When I run `wp user get 1 --field=user_pass`
+  Then STDOUT should not contain:
+    """
+    {ORIGINAL_PASSWORD}
+    """
+
+  @require-wp-4.3
+  Scenario: Reset the password of a WordPress user, and show only the new password
+    Given a WP installation
+
+    When I run `wp user get 1 --field=user_pass`
+    Then save STDOUT as {ORIGINAL_PASSWORD}
+
+    When I run `wp user reset-password 1 --skip-email --porcelain`
+    Then save STDOUT as {NEW_PASSWORD}
```

Comment:
TODO: Learn enough behat to have this check the output (it should be a password, currently "exactly 12 characters, no whitespace, etc")

Reply:
The check doesn't need to be super accurate. In other cases, we simply confirm that `STDOUT` is not empty.

Behat probably doesn't support checking "exactly 12 characters, no whitespace, etc". In order to perform such a check, you'd need to write a standalone PHP file to execute the check, pass the password to the PHP file, and then have the PHP file exit with 0 or 1 depending on whether it passed or failed. But, doing so is probably overkill for this particular test.